thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/template.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="row te-modal__settings">
+  <div class="col-md-10 no-padding">
+    <label for="select-metric" class="control-label te-label te-label--taller required">
+      Need support? Share the content below with the Thirdeye team at
+      <a class="thirdeye-link-secondary-grey" target="_blank" href="mailto:ask_thirdeye@linkedin.com">ask_thirdeye@linkedin.com</a>
```

Comment:
Can you configure and fetch this email (ask_thirdeye) from the environment.js file? That way, open source users also can customize this as per their help alias.